src/Profile/Profile.tsx

```diff
@@ -32,8 +31,7 @@ export const EDIT = "EDIT";
 function Profile() {
 
     useEffect(()=>{
-        //When login is merged, Authorization should be in Http.tsx, so we need to erase this "bearString" mock.
-        get(`user/getLogged`, {headers: {"Content-Type": "application/json","Authorization": `Bearer ${bearString}`}})
+        get(`user/getLogged`)
             .then(res => setSelectedProfile(res))
             .catch(err => alert(err.message));},[])
```

Comment:
Don't use the alert of the browser print into console instead 
`console.log(err.message)`